examples/js/loaders/GLTFLoader.js

```diff
@@ -1353,6 +1353,117 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
+	/**
+	 * Checks if we can build a single Mesh with MultiMaterial from the primitives.
```

Comment:
Can you please explain the reason for wanting to construct a single mesh when the primitives need to be rendered separately anyway?

Reply:
Yup, I'll do tomorrow. It's time to sleep here now.

Reply:
Thank you for the explanation. :)